addons/link_tracker/models/link_tracker.py

```diff
@@ -197,6 +196,27 @@ def search_or_create(self, vals):
 
         return self.create(vals)
 
+    @api.model
+    def search_or_create_multi(self, vals_list):
```

Comment:
Could be great that 'search_or_create' calls 'search_or_create_multi' so that we centralize computation